docs/adr/0012-output-markdown-format.md

```diff
@@ -0,0 +1,58 @@
+# 12. Output Markdown Format
+
+Date: 2021.09.09
+
+## Status
+
+proposed
+
+## Context
+
+Stencil has an output target that is capable of generating markdown documentation for the components that it compiles.
+Markdown does not have a unifying standard, which has led to multiple different implementations of how the markup is
+parsed. This has resulted in various markdown renderers whose implementations yield different renderings. Depending on
+where an individual views a markdown file, the results may look incorrect. 
+
+There was a [recent pull request](https://github.com/ionic-team/stencil/pull/3042/) that the team ultimately decided to
+close for not following CommonMark, which is the proposed/intended solution. Stencil should formally decide on a 
+proposed standard and use that for guidance as to how markdown should be formatted.
+
+## Options
+
+1. [Describe the many options, their pros/cons, side effects, and any perspective a given team member has]
+2. Adopt [CommonMark](https://commonmark.org/)
+   1. Pros:
+      1. It has industry support from the likes of GitHub, GitLab, Qt, and more
+      2. It seems to be the leading effort to create an unambiguous specification for Markdown
+      3. Using a well adopted standard may lead to less chance of Markdown renderings not working as intended
+   2. Cons
```

Comment:
Another major drawback of CommonMark is that it doesn't define a syntax for tables, so tables have to be produced as [HTML blocks](https://spec.commonmark.org/0.30/#html-blocks).

Reply:
Added in b59a91c1a7bcfa79be646ac0a464f455f07db070


To be fair, I dare say I prefer HTML tables in this scenario. I can never remember how to make a table in MD